integration/js/steps/WaitForAllParticipantsToJoinMeeting.js

```diff
@@ -0,0 +1,30 @@
+const AppWaitTestStep = require('../utils/AppWaitTestStep');
+
+class WaitForAllParticipantsToJoinMeeting extends AppWaitTestStep {
+  constructor(kiteBaseTest, sessionInfo, participantCount) {
+    super(kiteBaseTest, sessionInfo);
+    this.participantCount = participantCount;
+  }
+
+  static async executeStep(KiteBaseTest, sessionInfo, participantCount) {
+    const step = new WaitForAllParticipantsToJoinMeeting(KiteBaseTest, sessionInfo, participantCount);
+    await step.execute(KiteBaseTest);
+  }
+
+  stepDescription() {
+    return 'Waiting for all participants to join the meeting';
+  }
+
+  waitCompleteCondition() {
+    console.log("participantCount " + this.participantCount);
+    console.log("numberOfParticipant " + this.numberOfParticipant);
+    return this.numberOfParticipant.toString() === this.participantCount;
```

Comment:
I think we aready have a Wait step defined for this? Can we reuse that?

Reply:
yes, we already have a WaitForRemoteParticipantsToJoinMeeting, but somehow the waitCompleteCondition can not meet: `this.test.numRemoteJoined.toString() === this.numberOfParticipant`. In my case, the numRemoteJoined is 1 and the numberOfParticipant is 2. In order not to break any existing test that uses the WaitForRemoteParticipantsToJoinMeeting, I have to create a new step called WaitForAllparticipantsToJoinMeeting.